VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -117,6 +105,38 @@ public void onClick(View v) {
         }
     }
 
+    public static void setAuthorText(@NonNull TextView textView, @NonNull IAuthorData authorData) {
+        final CharSequence authorText;
+        {
+            final Context context = textView.getContext();
+            List<CharSequence> joinableStrings = new ArrayList<>();
+            final String author = authorData.getAuthor();
+            if (!TextUtils.isEmpty(author)) {
+                final SpannableString authorSpan = new SpannableString(author);
+                // Change the author text color and style
+                authorSpan.setSpan(new ForegroundColorSpan(
+                                context.getResources().getColor(R.color.edx_brand_primary_base)),
+                        0, authorSpan.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+                authorSpan.setSpan(new StyleSpan(Typeface.BOLD), 0, authorSpan.length(),
+                        Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+                joinableStrings.add(authorSpan);
+            }
```

Comment:
Since this section is being shared between this method and the `setAuthorAttributionText()` method, it might make sense to create a local private utility method to do it, that would be invoked from both of these methods.